src/modules/iotjs_module_process.c

```diff
@@ -146,10 +146,6 @@ JS_FUNCTION(Compile) {
 
   size_t size = 0;
   char* source = iotjs__file_read(filename, &size);
-  if (source == NULL || size == 0) {
```

Comment:
I think NULL is also needed to be checked :)

Reply:
See https://github.com/Rokid/ShadowNode/blob/master/src/iotjs_util.c#L30, the NULL means the file doesn't exist.